test_scripts/Polices/Policies_Security/001_ATF_P_PolicyTable_GeneralSDL_behavior.lua

```diff
@@ -0,0 +1,82 @@
+----  Name of requirement that is covered.
```

Comment:
@Aleksandriys  each script for each requirement should be in separate feature/ branch and commit!!!